manala.ohmyzsh/meta/main.yml

```diff
@@ -17,6 +17,7 @@ galaxy_info:
         - wheezy
         - jessie
         - stretch
+        - buster
   galaxy_tags:
     - system
-    - oh-my-zsh
+    - ohmyzsh
```

Comment:
Official package name is `Oh-My-Zsh`, keep `oh-my-zsh` tag

Reply:
Dashes are no longer authorized in tags, consequence of hyphen's gate